rllib/env/base_env.py

```diff
@@ -290,18 +298,31 @@ def get_unwrapped(self) -> List[EnvType]:
 _DUMMY_AGENT_ID = "agent0"
 
 
+@Deprecated(new="with_dummy_agent_id", error=False)
 def _with_dummy_agent_id(env_id_to_values: Dict[EnvID, Any],
                          dummy_id: "AgentID" = _DUMMY_AGENT_ID
                          ) -> MultiEnvDict:
     return {k: {dummy_id: v} for (k, v) in env_id_to_values.items()}
 
 
+@DeveloperAPI
```

Comment:
why make this @DeveloperAPI?

Reply:
Yeah, probably not necessary here. Let's not make this DevAPI.

Reply:
kk